app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -58,6 +58,12 @@ def textual_ext_management_system
     textual_link(ExtManagementSystem.find(@record.ems_id))
   end
 
+  def textual_physical_rack
+    rack_id = @record.physical_rack_id
+    return nil if rack_id.nil?
```

Comment:
You assign to a variable just for the test. The statement below again uses the right side of the assignment. This probably was not intended. Was it?

Reply:
You're right, fixed. :)